certs/pki/vault.go

```diff
@@ -106,8 +106,7 @@ func (p *pkiAgent) IssueCert(cn string, ttl, keyType string, keyBits int) (Cert,
 		return Cert{}, errors.Wrap(errFailedCertDecoding, err)
 	}
 
-	// Expire time calc must be revised
-	// value doesnt look correct
+	// Expire time calc must be revised value doesnt look correct
```

Comment:
Can you explain this? Can we remove this comment?

Reply:
@dusanb94  I had a problem here with converting response from vault API client to a structure, I used `mapstructure` lib since the response from `s, _ := api.ParseSecret(resp.Body)` in `s.Data` is `map[string]interface{}` , all fields were good but expiration. 
Now since we are going to remove vault api client with #1225 this piece should be refactored and since the response to be handled will be JSON i dont expect problem with converting value to appropriate data type
I will update issue #1225 not to forget to tackle this problem.